user/contactmethod/store.go

```diff
@@ -261,6 +271,12 @@ func (db *DB) CreateTx(ctx context.Context, tx *sql.Tx, c *ContactMethod) (*Cont
 
 	_, err = wrapTx(ctx, tx, db.insert).ExecContext(ctx, n.ID, n.Name, n.Type, n.Value, n.Disabled, n.UserID)
 	if err != nil {
+		if strings.Contains(err.Error(), "user_contact_methods_type_value_key") {
+			var u string
+			row := db.findUserID.QueryRowContext(ctx, n.Type, n.Value)
+			row.Scan(&u)
+			return nil, validation.NewFieldError("Value", fmt.Sprintf("Contact method already exists for that type and value for user %v", u))
```

Comment:
This should be the format of `contact method already exists for that type and value; USER_ID=user-id-here`